website/docs/help-and-support/troubleshooting-guide/widget-errors.md

```diff
@@ -1,38 +1,84 @@
 # Widget Errors
 
-This section will help you troubleshoot common widget errors on the Appsmith platform.
+This section would help you troubleshoot common widget errors on the Appsmith platform.
 
-## JSON Form Errors
+## JSON form errors
```

Comment:
The heading are given so that all the errors for that widget can be clubbed under it. For now, JSON form errors just has one error documented 'Source data exceeds`. This would grow as we add more errors.

Reply:
We can do the restructuring.

Reply:
yes, absolutely agree that the JSON Form heading should be there. My point was to club redundant text and reduce unnecessary sub-headings. 
<img width="854" alt="Screenshot 2022-11-26 at 09 48 24" src="https://user-images.githubusercontent.com/32227135/204072049-54e56900-6a41-438f-952f-8bbc46bc0571.png">

What do you think of the below structure?

## JSON form errors
You may see the below errors when working with a JSON Form widget.

### Source data field limit

**Error Message:** `Source data exceeds 50 fields. Please update the source data` 

This error is seen when you try to bind the query/API response with a lot of fields to the source data property.
Screenshot here

### Cause
...

### Solution
....